index.js

```diff
@@ -21,30 +20,64 @@ const TranslationReducer = require('./lib/broccoli/translation-reducer');
 
 module.exports = {
   name: 'ember-intl',
-  addonOptions: null,
+  opts: null,
   isLocalizationFramework: true,
 
   included() {
     this._super.included.apply(this, arguments);
 
-    let app = this._findHost();
-    this.app = app;
-    this.addonOptions = this.intlConfig(app.env);
+    this.app = this._findHost();
+    this.opts = this.getOptions(this.app.env);
+    this.locales = this.findLocales();
+    this.translationTree = this._buildTranslationTree();
```

Comment:
I may end up moving this to be called where needed (within both treeFor hooks vs on `included`)